cheroot/ssl/__init__.pyi

```diff
@@ -8,7 +8,13 @@ class Adapter(metaclass=ABCMeta):
     ciphers: Any
     context: Any
     @abstractmethod
-    def __init__(self, certificate, private_key, certificate_chain: Any | None = ..., ciphers: Any | None = ...): ...
+    def __init__(
```

Comment:
I wouldn't bother reformatting pyi stubs. I wanted them separately to potentially backport stuff to the older py2-supporting versions of Cheroot but now it'd be reasonable to just transfer them into the Python modules (as a separate change, of course).